radixdlt/src/main/java/com/radixdlt/consensus/liveness/ScheduledTimeoutSender.java

```diff
@@ -33,10 +34,11 @@
  */
 public final class ScheduledTimeoutSender implements FixedTimeoutPacemaker.TimeoutSender, PacemakerRx {
 	private static final Logger log = LogManager.getLogger();
-
+	private static final long LOGGING_INTERVAL = TimeUnit.SECONDS.toMillis(1);
 	private final ScheduledExecutorService executorService;
 	private final Subject<View> timeouts;
 	private final Observable<View> timeoutsObservable;
+	private volatile long nextLogging = 0;
```

Comment:
Doesn't need to be volatile as this class should only ever be accessed by a single thread.

Reply:
ok